WebApplication/Backend/Services/SurveyService.cs

```diff
@@ -0,0 +1,75 @@
+﻿using health_clinic_class_diagram.Backend.Model.Survey;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using WebApplication.Backend.Repositorys;
+using WebApplication.Backend.Util;
+
+namespace WebApplication.Backend.Services
+{
+    public class SurveyService
+    {
+        private SurveyRepository surveyRepository;
+        private ISurveyRepository isurveyRepository = new SurveyRepository();
```

Comment:
Same as in other 2 PRs, only one field of type ISurveyRepository named surveyRepository